Source/Edit tool scripts/EditShape.cs

```diff
@@ -9,12 +9,14 @@ public abstract class EditShape : MonoBehaviour
     {
         private Matrix4x4 worldToLocalMatrix;
 
+        protected virtual Transform ShapeTransform => transform;
```

Comment:
What is the advantage of defining this? I assume having a single point to change things. seems fine if so.
But shouldn't it be lower case if it's a private/protected value?

Reply:
Would it make sense to rename it to `ReferenceTransform` then?

Reply:
I reverted my change for now.